.gitignore

```diff
@@ -1,6 +1,10 @@
 artefacts/
 bin.*/
+boost/
+distribution/macos/AppIcon.iconset/icon_*.png
+distribution/macos/AppIcon.icns
```

Comment:
Can't this be generated in the build dir?

Reply:
Sure, the icns file might as well be. The `AppIcon.iconset` dir contains a Contents.json file, though. I suppose we could copy that over too, though.